src/helpers.h

```diff
@@ -21,6 +21,12 @@ static float lerp(float v0, float v1, float t) {
     return v0 + t * (v1 - v0);
 }
 
+static bool isValidID(QString id) {
```

Comment:
I think it would make sense to have it in the `Client` class.

Reply:
True, I've created a static method `isValidID` in `Client` that is used by Settings.cpp during the migration and a corresponding `hasValidID` method that just calls `isValidID` on the userID of the Client..